src/MuTalk-Tests/MTAnalysisFileLoggerTest.class.st

```diff
@@ -0,0 +1,23 @@
+Class {
+	#name : 'MTAnalysisFileLoggerTest',
+	#superclass : 'TestCase',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'tests' }
+MTAnalysisFileLoggerTest >> testLogFileExists [
+
+	| analysis logFileName logFile |
+	logFileName := 'logFileForTest.txt'.
+	logFile := logFileName asFileReference.
+	analysis := MTAnalysis new
+		            classesToMutate: { MTAuxiliarClassForMTAnalysis };
+		            testClasses: { MTAuxiliarClassForMTAnalysisTest };
+		            logger: (MTAnalysisFileLogger toFileNamed: logFileName).
+
+	analysis run.
+
+	self assert: logFile exists.
+	logFile delete
```

Comment:
Hi @DurieuxPol it should be better to put the delete in a `tearDown` method.
Otherwise, if the test fails for some reason, the system will remain dirty. (and the test will never pass then)